### ISSUE TITLE
use correct focusConfig for features that uses the cxFocus directive

### ISSUE THREAD

**Issue by bgambocjaviniar** [SAP/spartacus#11805]
Currently known features:

- [ ] saved cart form dialog from cart page
- [ ] anonymous consents

Context: 
- need to use the correct focus config to display the focus ring (the blue outline when tabbing).

Example:
- when we type in the saved cart dialog from the cart page, it appends the 'mouse-focus' class to the skip link's `main` element. But when we navigate out of the inputbox/textarea, it should remove the 'mouse-focus' class in order to display the focus ring. Basically, when we type, it should not have the focus ring, but when we tab to another element, the focus ring should be back.

**NOT A BLOCKER as discussed with Miguel**

**Comment by developpeurweb** [comment]
- An alternative solution to this issue is to LEAVE the focus ring while interacting (typing or selecting). 
- The focus ring **shouldn't necessarily be removed** while using the keyboard. Only when using the mouse is when we remove it.
- Also please notice, this does NOT HAPPEN with the **add-to-cart modal** for some reason e.g. Quantity field always has a focus ring

Bug example:
1. While using the keyboard, the class attribute is **empty**, therefore the focus ring is shown.

![2021-04-13_13-08-17](https://user-images.githubusercontent.com/15113219/114593095-b5242780-9c59-11eb-9ac9-462b4df34d9f.jpg)

2. As soon as the user interacts with the form element (checkbox), the `mouse-focus` class kick-in and the focus ring becomes invisible.

![Screen Shot 2021-04-13 at 1 08 56 PM](https://user-images.githubusercontent.com/15113219/114593514-2c59bb80-9c5a-11eb-9bb9-aa99a854c962.png)
---
The same thing happens with **Saved Cart**

<img width="1031" alt="Screen Shot 2021-04-01 at 12 45 04 PM" src="https://user-images.githubusercontent.com/15113219/114593724-67f48580-9c5a-11eb-9712-9e70823ad2f5.png">
